app/models/work.rb

```diff
@@ -748,7 +748,10 @@ def chapter_total_display
   # Set the value of word_count to reflect the length of the chapter content
   def set_word_count
     if self.new_record?
-      self.word_count = self.chapters.first.set_word_count
+      self.word_count = 0
```

Comment:
I initially thought not as the original method relies on the chapters being in the database, which they're not in this `before_save` method if the work is brand new (has never been saved). Using only the code I've added, which uses the counts from the chapters associated with the work in memory, made one of the chapter tests fail, but that turned out to be because it was possible for a chapter to be saved with a nil word count. Once I fixed _that_ bug, both creating and editing works appears to return the right word counts. So once I've written some more tests around word counts, as I don't think we're covering everything, I think this will be ready to go. (I've set to Needs coder action)